packages/expo-polyfills/src/index.ts

```diff
@@ -210,7 +210,15 @@ class SyncSecureStorage implements Storage {
             })
             .catch(reject);
         })
-        .catch(reject);
+        .catch((err: Error) => {
+          console.error(err);
+          SecureStore.deleteItemAsync('___keys___')
```

Comment:
👍 

I don't know if it's worth trying to catch this one specific error?  Probably not worth the risk of false positives / false negatives.

Reply:
What do you mean exactly? Which specific error? Or what is the alternative

Reply:
Oh I think I understand, like looking at the `error.message` for only this error?

Yeah I think if `getItemAsync` fails it's more robust just to start over 👍